t/fuzz/fuzz_parseini.c

```diff
@@ -0,0 +1,37 @@
+#include "fio.h"
+
+static int initialized = 0;
+
+int LLVMFuzzerTestOneInput(const uint8_t* data, size_t size) {
+    if (size < 2) {
```

Comment:
We prefer single line condition bodies not to have braces

Reply:
ok